MyOnlineStore/ruleset.xml

```diff
@@ -103,4 +103,10 @@
             <property name="spacesCountBeforeColon" value="0"/>
         </properties>
     </rule>
+
+    <rule ref="SlevomatCodingStandard.TypeHints.UnionTypeHintFormat">
+        <properties>
+            <property name="withSpaces" value="yes" />
```

Comment:
```suggestion
            <property name="withSpaces" value="yes" />
            <property name="shortNullable" value="no" />
            <property name="nullPosition" value="last" />
```

I'm not entirely sure if this is merged with the defaults in the Doctrine standard, or if this is overwritten. In case of the latter, we should add these properties as well.

Reply:
Tested this by locally adding this change to the monolith config, followed by adding a short nullable in the code somewhere. PHPCS still complained about it :smile: 

![Screenshot from 2022-11-18 16-40-43](https://user-images.githubusercontent.com/6802443/202743559-d32dc057-0781-4ff5-b37f-cc2ed5b8c5be.png)
![Screenshot from 2022-11-18 16-40-19](https://user-images.githubusercontent.com/6802443/202743566-bf3acd01-7aee-42ca-b294-5efc4fb9d3df.png)